railties/lib/rails/generators/rails/app/templates/Dockerfile.tt

```diff
@@ -49,6 +51,7 @@ RUN SECRET_KEY_BASE_DUMMY=1 ./bin/rails assets:precompile
 
 <% end -%>
 
+# Final stage for app image
 FROM base
 
 # copy built artifacts: libraries, gems, application
```

Comment:
Should capitalize Copy to fit with the rest of the comment style.

Reply:
Fixed.